### PR TITLE
NodeJS: Strip HTML from output

### LINKED ISSUE #591: NodeJS: Need to remove HTML from parsed documentation content
### Description

https://duckduckgo.com/?q=http.Server.checkExpectation&ia=about

Looking at the output.txt I can see we're pulling the HTML straight from the docs, which already contains `<p>` and `<a>` tags. The `<a>` tags are known to cause issues and should be removed.

By wrapping the `description` in `<p>` we're breaking the pre-existing `<p>` tags, which is causing the browser to add additional `<p>` tags, creating a lot of extra whitespace.

We need to remove all `<p>` and `<a>` tags from the content before using it. We can keep the `<code>` tags if we want, but simply removing all HTML from the parsed content will be the easiest solution here.

## People to notify
@bfmags @jarrodthibodeau 

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/node_js

### PR DESCRIPTION
**Pull request by willyb321**
## Description of new Instant Answer, or changes

Strips all html tags from the answer.

## Related Issues and Discussions

Fixes #591 

## People to notify

@bfmags @jarrodthibodeau @moollaza 

---

Instant Answer Page: https://duck.co/ia/view/node_js

### CODE REVIEW

**Comment by daxtheduck** [comment]
@willyb321 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by willyb321** [comment]
For a comparison:  
[My changes](http://i.imgur.com/7hyX7Nn.png)  
[Current Live](http://i.imgur.com/4RW5emA.png)

**Comment by moollaza** [comment]
@willyb321 Thanks! This LGTM :+1: Excellent work 👍 💯  

BTW, the first thing I noticed when I looked at the output.txt is that the first few articles are all duplicated. Would you be interested in fixing that as well?

Thanks again 👍

File: lib/fathead/node_js/parse.js [hunk 1]
```diff
@@ -33,28 +33,32 @@ function getAbstract(obj) {
...
-
     // there's some non-utf8 stuff that postgres doesn't like:
-    a = a.replace(/0x00|\\0|\\u0000/g,' ');
-
+    a = a.replace(/0x00|\\0|\\u0000/g,'');
+    
     // would be nice to be able to link these, but the links seem to get stripped out:
     a = a.replace(/\[(.*)\]\[\]/g,'$1');
-
+
```
Comment (moollaza) [inline lib/fathead/node_js/parse.js]: Looks like you accidentally added some trailing spaces?

**Comment by willyb321** [comment]
@moollaza I could probably fix that, would you like me to open another PR for it or ?

**Comment by moollaza** [comment]
> @moollaza I could probably fix that, would you like me to open another PR for it or ?

Yes please!

**Comment by tagawa** [comment]
Hi @willyb321. Just wanted to let you know this is now live on DuckDuckGo!
https://duckduckgo.com/?q=http.Server.checkExpectation&ia=about

Thanks again for improving this for NodeJS developers 😃

**Pull request merged**